codelists/codelists.txt

```diff
@@ -1,3 +1,27 @@
 opensafely/chronic-cardiac-disease/2020-04-08
 opensafely/chronic-liver-disease/2020-06-02
 opensafely/asthma-inhaler-salbutamol-medication/2020-04-15
+opensafely/rheumatoid-arthritis/2020-05-12
+opensafely/asthma-oral-prednisolone-medication/2020-04-27
+opensafely/cancer-excluding-lung-and-haematological/2020-04-15
+opensafely/chronic-cardiac-disease/2020-04-08
+opensafely/chronic-kidney-disease/2020-04-14
+opensafely/chronic-liver-disease/2020-06-02
+opensafely/chronic-respiratory-disease/2020-04-10
+opensafely/current-copd/2020-05-06
+opensafely/covid-identification/2020-06-03
+opensafely/covid-identification-in-primary-care-probable-covid-clinical-code/2020-07-16
+opensafely/covid-identification-in-primary-care-probable-covid-positive-test/2020-07-16
```

Comment:
comment elsewhere but currently we just have sgss in protocol

Reply:
We will remove